### PR TITLE
Remote Joins: Create relationships across database and remote schemas

### PR DESCRIPTION
**Pull request by wawhal**
This is a stale PR. Checkout #2392

### CODE REVIEW

File: console/src/components/Services/Data/DataActions.js [hunk 1]
```diff
@@ -264,10 +266,13 @@ const loadSchema = configOptions => {
         fetchTrackedTableListQuery(configOptions), // v1/query
         fetchTrackedTableFkQuery(configOptions),
         fetchTrackedTableReferencedFkQuery(configOptions),
-        fetchTrackedTableRemoteRelationshipQuery(configOptions),
       ],
     };
 
+    if (featuresCompatibility.RemoteRelationShips) {
```
Comment (wawhal) [inline console/src/components/Services/Data/DataActions.js]: Use the constant `FT_REMOTE_RELATIONSHIPS` exported from `src/helpers/versionUtils`.

File: console/src/components/Services/Data/DataActions.js [hunk 2]
```diff
@@ -277,12 +282,21 @@ const loadSchema = configOptions => {
 
     return dispatch(requestAction(url, options)).then(
       data => {
+        const tableList = JSON.parse(data[0].result[1]);
+        const fkList = JSON.parse(data[2].result[1]);
+        const refFkList = JSON.parse(data[3].result[1]);
+        let remoteRelationships = [];
+
+        if (featuresCompatibility.RemoteRelationShips) {
```
Comment (wawhal) [inline console/src/components/Services/Data/DataActions.js]: `FT_REMOTE_RELATIONSHIPS`

**Comment by praveenweb** [comment]
@wawhal - The remote schema list in relationships section doesn't subscribe to changes in the original remote schema list.

**Comment by c19** [comment]
Where can I donwload matched version of hasura cli?
Can't find on docker hub.

**Comment by niklabh** [comment]
Where can we get version which applies migrations automatically

**Comment by webdeb** [comment]
@wawhal it is not possible to use this preview container with an existing 1.0.0 instance.
The error is something like: Cannot use a version less then the previous.. Is it possible to tag this preview with a version `>1.0.0` ? Thank you

**Comment by joni7777** [comment]
Any updates on when to expect this to be delivered? Really need this amazing feature! 

Thanks

**Comment by weyert** [comment]
Looking forward to this feature :)

**Comment by picosam** [comment]
@tirumaraiselvan how will this change (if at all) now that Actions support queries as well?

**Comment by weyert** [comment]
Any chance master can be merged into this PR?

**Comment by ElixirMike** [comment]
Is this close to being ready for release? Any timeframe you can provide?

**Comment by tirumaraiselvan** [comment]
Remote Joins has been merged in https://github.com/hasura/graphql-engine/pull/2392

**Pull request closed without merging**